addons/openebs/1.12.0/install.sh

```diff
@@ -256,3 +259,149 @@ function openebs_cstor_pool_count() {
     local actual=$(kubectl -n "$OPENEBS_NAMESPACE" get cstorpools -l openebs.io/storage-pool-claim=cstor-disk --no-headers 2>/dev/null | wc -l)
     [ $actual -ge $target ]
 }
+
+function openebs_upgrade() {
+    # TODO: handle namespace spec changes at the upgrade time
+    if [ "$OPENEBS_CSTOR" = "0" ]; then
+        # upgrades only required for cStor 
+        return 0
+    fi
+
+    # RE: https://github.com/openebs/openebs/tree/master/k8s/upgrades/1.x.0-1.12.x
+    # Upgrading a minor version of 1.x.x requires two jobs to update pools a volumes for cStor configurations.
+    local runningVer=$(kubectl -n $OPENEBS_NAMESPACE get deploy openebs-provisioner -o jsonpath='{.metadata.labels.openebs\.io/version}')
+
+    local semVerRunningList=( ${runningVer//./ } )
+    local semVerInstallList=( ${OPENEBS_VERSION//./ } )
+    
+    if [[ ${semVerInstallList[0]} -gt 1 ]]; then
+        bail "Only upgrades upto OpenEBS 1.12.0 are tested and supported."
+    fi
+
+    logSubstep "Runnig upgrade checks..."
+    if [[ ${semVerInstallList[1]} -gt ${semVerRunningList[1]} || openebs_check_pools ]]; then
+        logSubstep "As part of OpenEBS upgrade pools and volumes need to be upgraded. At least some of the pools and(or) volumes require an update to match OpenEBS controll plane."
+        logFail "It is highly recommended to schedule a downtime for the application using the OpenEBS PV while performing this upgrade. Also, make sure you have taken a backup of the data before starting the below upgrade procedure."
+        printf "Continue? "
+        if ! confirmN " "; then
+            bail "Will not upgrade OpenEBS. Modify your spec and re-run instllation."
+        fi
+
+        openebs_upgrade_pools
+        openebs_upgrade_vols
```

Comment:
we should wait for these jobs to all reach the completed state with a spinner before continuing

Reply:
I added a comment on that https://github.com/replicatedhq/kURL/blob/07c01296778162ea03b1d99eaa4bd83765f94c17/addons/openebs/1.12.0/install.sh#L341

It might take a while in some cases, I wasn't sure if we want to block on those jobs.

Reply:
but will other things work while the migration is running? If not, we should block

Reply:
Continuing the installation will work, so there is no hard need to block. What we would miss is to provide feedback if those jobs completed successfully, however.